src/serac/physics/materials/solid_material.hpp

```diff
@@ -152,7 +152,7 @@ struct J2 {
  * @tparam dim number of spatial dimensions
  *
  * @param displacement_gradient Displacement gradient
- * @param kirchhoff_stress Kirchoff stress
```

Comment:
doh! I wrote this one. Got the name right in the variable but wrong in the comment. Probably copied and pasted it like a drone.

Reply:
Well, considering my terrible spelling is probably to blame I can't get too mad. 😆